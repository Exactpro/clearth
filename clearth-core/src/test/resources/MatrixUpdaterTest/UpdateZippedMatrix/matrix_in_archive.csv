#Id,#GlobalStep,#Action,#Character,#Code
id1,Step1,SetStatic,Z,9901
#Id,#GlobalStep,#Action,#Code
id2,Step1,SetStatic,9901
