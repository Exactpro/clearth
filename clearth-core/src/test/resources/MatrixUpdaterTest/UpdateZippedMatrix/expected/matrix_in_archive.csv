#Id,#GlobalStep,#Action,#Character,#Code,#AnotherCharacter
id1,Step1,SetStatic,Z,9901,A
#Id,#GlobalStep,#Action,#Code,#AnotherCharacter
id2,Step1,SetStatic,9901,A
